cn/localisation/l_english.yml

```diff
@@ -3550,7 +3550,7 @@
  # Ship Designer
  ####################################
 
- COMPONENT_SET_MULTIPLE_SIZES:0 "悬停于大小图标以查看详细信息"
+ COMPONENT_SET_MULTIPLE_SIZES:0 "悬停于图标上以查看详细信息"
```

Comment:
了解，当时就是看这个大小图标说的别扭

发自我的 iPhone

在 2016年8月9日，23:17，云风 <notifications@github.com<mailto:notifications@github.com>> 写道：


In cn/localisation/l_english.yml<https://github.com/cloudwu/stellaris_cn/pull/4#discussion_r74079356>:

> @@ -3550,7 +3550,7 @@
>   # Ship Designer
>   ####################################
>
> - COMPONENT_SET_MULTIPLE_SIZES:0 "悬停于大小图标以查看详细信息"
> + COMPONENT_SET_MULTIPLE_SIZES:0 "悬停于图标上以查看详细信息"


这里原文是 size icon, 真的是大小图标（舰船配件分大中小）哦。在舰船设计界面上的一个提示。

―
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/cloudwu/stellaris_cn/pull/4/files/23b045888fbc6974053dee430a1e4b186b8c893d..632302e18772959a5db3b1d669946f9563c8214b#r74079356>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABIbHAZjihAZsjpP6COL5t5uY1nsthlIks5qeJn0gaJpZM4Jf6Rz>.